src/components/RecipesForm.js

```diff
@@ -0,0 +1,114 @@
+import { useState } from "react"
+import { TextField, Button } from "@mui/material"
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select from '@mui/material/Select';
+import '../styles/RecipesForm.css'
+
+const RecipesForm = ({reloadList, setReloadList}) => {
+    
+    const [recipeName, setRecipeName] = useState('')
+    const [cuisineType, setCuisineType] = useState('')
+    const [estTimeOfPrep, setEstTimeOfPrep] = useState('')
+    const [ingredients, setIngredients] = useState('')
+    const [prepInstructions, setPrepInstructions] = useState('')
+
+    const handleRecipesForm = (event) => {
+        event.preventDefault()
+        if (recipeName && cuisineType && estTimeOfPrep && ingredients && prepInstructions) {
+            const recipeObject = {
+                recipeName,
+                cuisineType,
+                estTimeOfPrep,
+                ingredients,
+                prepInstructions
+            }
+            fetchCreateRecipe(recipeObject)
+        }
+    }
+
+    const handleChange = (event) => {
+        setCuisineType(event.target.value);
+    };
+    
+    const fetchCreateRecipe = async (recipe) => {
+
+        const token = localStorage.getItem('token')
+        const userId = localStorage.getItem('user')
+
+        const url = `https://recipe-blog-l7ey.onrender.com/recipe-blog/mypage`
+
+        const options = {
+            method: 'POST',
+            headers: {
+                Authorization:`Bearer ${token}`,
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify({
+                "recipeName": recipe.recipeName,
+                "createdBy": userId,
+                "cuisineType": recipe.cuisineType,
+                "estTimeOfPrep": recipe.estTimeOfPrep,
+                "ingredients": recipe.ingredients,
+                "prepInstructions": recipe.prepInstructions
+            })
+        }
+
+        try {
```

Comment:
In this code, you need to check whether the API call was successful or not. If it was successful, you have to set the reload list. If it wasn't successful, you have to set the error message. 

Here's the code:

```
try {
  const response = await fetch(url, options);
  const data = await response.json();

  if (response.ok) {
    setReloadList(reloadList + 1);
  } else {
    setErrorMessage(data.message);
  }
} catch (error) {
  setErrorMessage("Request failed");
}
``` 

Let me know if you need any further assistance.